automation/scripts/gen-keys-ledger.sh

```diff
@@ -1,5 +1,5 @@
 #! /bin/bash
-MINA_DAEMON_IMAGE="minaprotocol/mina-daemon:1.3.2beta2-release-2.0.0-6f9d956-focal-berkeley"
+MINA_DAEMON_IMAGE="minaprotocol/mina-daemon:1.3.2beta2-release-2.0.0-6f9d956-focal-devnet"
```

Comment:
There is no such image as ...-devnet . This is an old image, which leads to a conclusion that it is a old script (not anymore valid). I would revert this change

Reply:
i see, reverting then